src/javascript/static/pages/responsible_trading.js

```diff
@@ -0,0 +1,26 @@
+const State          = require('../../_common/storage').State;
+const Client         = require('../../app/base/client');
+const BinarySocket   = require('../../app/base/socket');
+const getElementById = require('../../../javascript/_common/common_functions').getElementById;
```

Comment:
why not follow the same path as `State`?
```suggestion
const getElementById = require('../../_common/common_functions').getElementById;
```

Reply:
Just a mistake. Fixed